documentation/modules/security/con-kafka-listener-certificates.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='kafka-listener-certificates-{context}']
+
+= Kafka listener certificates
+
+In order to connect to external systems such as {ProductName}, your client applications might need to trust server certificates that were issued by an external CA, such as your company's private CA or a public CA.
```

Comment:
I do not think this sentence makes sense. I think it should say something like this:
_Sometimes, you might want to use {ProductName} with your own CA such as  your company's private CA or a public CA._
or maybe:
_Sometimes you want to leverage your existing security infrastructure such as your company's private CA or a public CA._

Reply:
Very nice. I feel that the verb _leverage_ demands an action. 

> Sometimes you might want to leverage your existing security infrastructure, such as your company's private CA or a public CA, when integrating your client applications with {ProductName}. 

I'll remove the whole sentence beginning with "In order to...".